.travis.yml

```diff
@@ -16,8 +16,11 @@ jobs:
     - name: Linux | arm64 | Python 3.6
       python: 3.6
       services: docker
-      arch: arm64
+      arch: arm64-graviton2
+      group: edge
+      virt: vm
       env: PYTHON=python
+      allow_failure: True
```

Comment:
Wasn't the `allow_failure` meant to be added on ppc64le ?

Reply:
😳